packages/cashier/package.json

```diff
@@ -48,7 +48,6 @@
         "loadjs": "^4.2.0",
         "lodash.debounce": "^4.0.8",
         "mobx": "^6.6.1",
-        "mobx-react": "^7.5.1",
```

Comment:
Should’ve been `mobx-react-lite`, But seems like we have missed it in merge conflicts and didn’t notice it.